inst/htmlwidgets/dygraphs.js

```diff
@@ -777,7 +777,7 @@ HTMLWidgets.widget({
       },
       
       // export dygraph so other code can get a hold of it
-      dygraph: dygraph
+      dygraph: null
```

Comment:
Why exporting `dygraph` as `null`?

Reply:
not exporting as `null` rather intialiazing as `null` which gets populated on render in https://github.com/timelyportfolio/dygraphs_htmlwidget/blob/b1834d8608c011d210e79cfedcd259e27efcec20/inst/htmlwidgets/dygraphs.js#L231.  I should note a function version would work here if you prefer.  This is similar to leaflet's `getMap()` ([lines](https://github.com/rstudio/leaflet/blob/master/inst/htmlwidgets/leaflet.js#L518-L521)) but might surprise dygraphs users expecting an `object`.

```
dygraph: function() {return dygraph;}
```